src/printer/scalebar.js

```diff
@@ -125,27 +123,19 @@ function getScaleBarParams(frameState, spec) {
  * @param {PrintSpec} spec
  */
 function renderScaleBar(ctx, frameState, scaleBarParams, spec) {
-  const mapScale = `1 / ${spec.scale}`;
-
   const scaleWidth = scaleBarParams.width;
   const scaleNumber = scaleBarParams.scalenumber;
   const scaleUnit = scaleBarParams.suffix;
 
   const scaleText = `${scaleNumber} ${scaleUnit}`;
   const scaleTextWidth = ctx.measureText(scaleText).width;
 
-  const scaleTitle = (spec.scaleBar.template
-    ? spec.scaleBar.template
-    : DEFAULT_TITLE
-  ).replace('{mapScale}', mapScale);
-  const scaleTitleWidth = ctx.measureText(scaleTitle).width;
-
```

Comment:
Just saying, 'template' was never described as part of the ScaleBarSpec in the README.

Reply:
Oups, you're right, another reason to get rid of it ;-)